taskcluster/.shared.yml

```diff
@@ -1,8 +1,8 @@
 python:
   packages_trusty:
-    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
+    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libhdf5-dev'
```

Comment:
So I guess you did that because of the failed Python tests. However, I'm not sure we want to have to build `tables` from source, it might delay quite a lot the process. And the lack of 3.4 wheel might be an upstream issue.

I've opened https://github.com/mozilla/DeepSpeech/pull/1961 and https://github.com/PyTables/PyTables/issues/727, it might be better we don't pull `libhdf5-dev` then.

Reply:
@reuben So I'd suggest we merge #1961 and remove this commit :)